src/plugins/internal/InternalInstallPlugin.ml

```diff
@@ -146,9 +77,12 @@ let install pkg argv =
       fun fn -> fn
   in
 
-  let install_file ?tgt_fn src_file envdir =
+  let install_file ?(with_destdir=true) ?tgt_fn src_file envdir =
```

Comment:
So if I understand correctly the whole change, in order to circumvent 1 problem of ocamlfind install (inability to install in subdirectories), you disable it completely?

What about a more conservative approach like:
- if no directory defined, just use standard procedure (ocamlfind install...)
- if a directory is defined:
  1 'ocamlfind install library META [file going into the root directoryof the install]'
  2 'ocamlfind query library.subpackage' -> will print where files for library.subpackage should be installed
  3 install your extra data into the result of 2
  4 restart at 2 for all subpackages with a directory